### DISCUSSION TITLE
install via dnf on Rocky Linux

### DISCUSSION THREAD

**Discussion by fahadshery** [timescale/timescaledb-toolkit#374] [Q&A]
Hi,

I already have `timescaledb` and `postgres` installed on my `red-hat` based `Rocky Linux`. Is there a way I could `dnf` install this extension?

Thanks

**Comment by jerryxwu** [comment]
@fahadshery have you tried the instructions documented [here](https://docs.timescale.com/timescaledb/latest/how-to-guides/hyperfunctions/install-toolkit/#install-toolkit-on-self-hosted-timescaledb)?
